README.md

```diff
@@ -1 +1,159 @@
-# fe-hometask
\ No newline at end of file
+# React & Rockets
+
+**Description:** Challenge for Adjust Frontend Developer
+
+**Owner:** [adjust](https://github.com/adjust)
+
+**Contacts:** frontend-hiring@adjust.com
+
+---
+
+## Welcome to our little coding exercise! 👋
+
+Here you will be given the opportunity to play with JavaScript and Rockets in the same project. For this, we recommend you to block 2-3 hours of your time to complete it.
+
+**Please make sure you have read this page entirely, before starting the challenge**. If you have any doubts, please reach out to us by [opening an issue](#how-to-request-help) as described at the bottom of this page.
+
+**Importantly**, we would like to give all candidates taking this test the same opportunity to solve the exercise in their own way. Because of this, **please do not fork or share this repo (nor your solution) with anybody 🙏🏻**
+
+<img align="center" src="https://i.imgur.com/ekyJNd9.jpg" width="600">
+
+## Instructions
+
+1. Push your solution to a **private repo** in your **personal Github account**
+2. When you are ready for us to take a look, please add [adjust-frontend-hiring][adjust-frontend-hiring] (GitHub user) as collaborator
+
+## General notes
+
+- The final solution must be a working React application.
+  - It's up to you how to bootstrap a new React application.
+- You can use any third-party library if you see a good fit for it.
+- It's not necessary to heavily comment the code, please only leave comments where they are necessary for code comprehension. However, we welcome notes and comments that reflect the higher-level decisions made during the challenge.
+- Aspects to be assessed:
+  - Code Style – meaningful naming conventions, consistent code formatting, and emphasis on code readability and future extensibility.
+  - Quality Assurance – the solution you upload should work as expected, conforming to the specifications of the tasks.
+  - Introducing best practices such as tests, type safety, and linting is appreciated.
+
+Good luck and happy coding! 🚀
+
+---
+
+## Exercise
+
+### TASK #1 - JavaScript
```

Comment:
Why we need a focus on JavaScript? What about to said it like TypeScript?

Because jQuery is also JavaScript, while EcmaScript & TypeScript can make more context

Reply:
Our general consensus in the job description is that TypeScript is nice to have, and we consider candidates who do not have a lot of experience with TypeScript as it can be picked up quite fast. So what we really want to see in the first task is that the candidates are comfortable working with JS.

Reply:
FYI, I've recently updated our [job profile for senior](https://docs.google.com/document/d/1YRO8S-afTm2gc9N8LaFg5ptYal-9hkx9IIwkWp5AW_I/edit#heading=h.y3s2vkxnp13u) moving typescript from 'nice to have' to 'required' as times are changing and I don't think it's realistic that we'll see senior candidates w/o a typescript experience.

Do we want to refactor home task because of this? Idk, we need the first version by the end of the week, but we can iterate later

Reply:
> Do we want to refactor home task because of this?

Wouldn't hurt exploring that. I will later look into how a solution with TypeScript could look like. Once we have that, we can get back to hometask specification and update that.

Thanks for sharing your inputs in this thread, I'll resolve it for now and notify in Slack hiring channel about TS addition to the spec.